src/common/memory_desc_wrapper.hpp

```diff
@@ -149,30 +149,28 @@ struct memory_desc_wrapper : public c_compatible {
     size_t additional_buffer_data_size(uint64_t flag_select) const {
         using namespace memory_extra_flags;
         if (flag_select & compensation_conv_s8s8) return sizeof(int32_t);
-        if ((flag_select & rnn_u8s8_compensation)
-                && !types::extra_flag_rnn_s8s8_compensation_is_set(flag_select))
-            return sizeof(float);
+        if (flag_select & rnn_u8s8_compensation) return sizeof(float);
         if (flag_select & compensation_conv_asymmetric_src)
             return sizeof(int32_t);
+        if (flag_select & compensation_gpu_conv_asymmetric_src)
+            return sizeof(int32_t);
```

Comment:
Do we need to handle `compensation_gpu_conv_asymmetric_src_bwd` here as well?
As a related comment: since `memory_extra_desc_t ` already has `*dhw` members which are only used by the GPU reorder implementation - we could add another field to indicate whether it's src/src_bwd/src_swap. Exposing this via two extra flags may be too much detail for the top level.